core/src/main/scala/org/apache/spark/rdd/RDD.scala

```diff
@@ -812,11 +813,13 @@ abstract class RDD[T: ClassTag](
    */
   private[spark] def mapPartitionsWithIndexInternal[U: ClassTag](
       f: (Int, Iterator[T]) => Iterator[U],
-      preservesPartitioning: Boolean = false): RDD[U] = withScope {
+      preservesPartitioning: Boolean = false,
+      orderSensitiveFunc: Boolean = false): RDD[U] = withScope {
```

Comment:
nit: add param comment for `orderSensitiveFunc`

Reply:
done